src/features/Students/StudentsMetrics/index.jsx

```diff
@@ -2,28 +2,34 @@ import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Card, CardGrid, ToggleButton } from '@edx/paragon';
 import { ToggleButtonGroup } from 'react-paragon-topaz';
-import { fetchMetricsData } from 'features/Students/data/thunks';
+import { fetchClassesMetricsData, fetchStudentsMetricsData } from 'features/Students/data/thunks';
 
 import 'features/Students/StudentsMetrics/index.scss';
 
 const StudentsMetrics = () => {
   const dispatch = useDispatch();
-  const stateMetrics = useSelector((state) => state.students.metrics.data);
+  const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
+  const stateStudentsMetrics = useSelector((state) => state.students.studentsMetrics.data);
+  const stateClassesMetrics = useSelector((state) => state.students.classesMetrics.data);
+  const days = 100; // Temporal variable
```

Comment:
I suggest to move this variable outside of this component in order to avoid re-calculate it in each render ( yes not a huge calculus but anyway... )